src/main.rs

```diff
@@ -561,6 +561,11 @@ fn start_vmm(cmd_arguments: ArgMatches) -> Result<Option<String>, Error> {
         .ok();
     }
 
+    // SAFETY: Ignoring a signal is async-safe.
```

Comment:
Does it matter whether it's async-safe?  Nothing async is happening at this point, right?

Reply:
Ah sorry, what I meant here was: the reason signal() is unsafe is that you might pass it an async-unsafe handler function, but the SIG_IGN action is trivially async-safe.

Reply:
The reason it's unsafe is just that it's a libc function, and they're all marked unsafe.  So I don't think it makes sense to call attention to this specific detail of it — there are all sorts of reasons a libc function _might_ be unsafe.  I think it'd make more sense just to say "Trivially safe" for this one, like we do for other simple syscalls.

Reply:
Okay, I've changed this.